### ISSUE TITLE
SSHNP E2E Test Flaking

### ISSUE THREAD

**Issue by xavierchanth** [atsign-foundation/noports#276]
### Describe the bug

On occasion the device name is not sent which causes the test to fail.

### Steps to reproduce

N/A

### Expected behavior

N/A

### Additional context

```
sshnp   | Device "562875661810" unknown, or username not shared 
sshnp   | Exception: Internal server exception : Request to remote secondary @8052simple at null:null received error response 'AT0015-Exception: @8incanteater:username.562875661[81](https://github.com/atsign-foundation/sshnoports/actions/runs/5628756618/job/15252980763#step:16:82)0.sshnp@8052simple does not exist in keystore'
sshnp   | 
sshnp   | Stack Trace: #0      AtClientImpl.get (package:at_client/src/client/at_client_impl.dart:349)
sshnp   | <asynchronous suspension>
sshnp   | #1      SSHNPImpl.fetchRemoteUserName (package:sshnoports/sshnp/sshnp_impl.dart:420)
sshnp   | <asynchronous suspension>
sshnp   | #2      SSHNPImpl.init (package:sshnoports/sshnp/sshnp_impl.dart:295)
sshnp   | <asynchronous suspension>
sshnp   | #3      main (file:///app/repo/packages/sshnoports/bin/sshnp.dart:26)
sshnp   | <asynchronous suspension>
```
[Full log](https://github.com/atsign-foundation/sshnoports/actions/runs/5628756618/attempts/1)

**Comment by JeremyTubongbanua** [comment]
Happening because the `default: 4` in the action.yaml for setting up the entrypoint is not passing when calling the script

![image](https://github.com/atsign-foundation/sshnoports/assets/79019866/befa0669-40b1-42cf-9e3b-cf8e96b28987)

Should be `./setup-sshnp-entrypoint.sh 59... @8incanteater @8052simple @8485wealthy51 4`

**Comment by JeremyTubongbanua** [comment]
#280 should fix the flaking

**Comment by JeremyTubongbanua** [comment]
Closed via #280
